components/email/src/init.spec.js

```diff
@@ -740,3 +1232,192 @@ describe("Email: Toggle email of sender/recipient", () => {
     });
   });
 });
+
+describe("Should Render Reply Button And Dispatch Event When Clicked", () => {
+  beforeEach(() => {
+    cy.visit("/components/email/src/cypress.html");
+
+    cy.get("nylas-email").as("email");
+
+    cy.get("@email").invoke("attr", "show_reply", "true");
+  });
+
+  it("Should Render Reply Button When Passed A Message", () => {
+    cy.get("@email").then((elements) => {
+      const component = elements[0];
+      component.message = SINGLE_SENDER_MESSAGE;
+      cy.get("@email").find("div.reply button").should("exist");
+    });
```

Comment:
In order to maintain the best practices we recently put in place, suggesting the following for better readability.

```suggestion
      cy.get("@email").invoke('prop', 'message', SINGLE_SENDER_MESSAGE);
      cy.get("@email").find("div.reply button").should("exist");
```